docs-src/typescript/how-to-emit-metrics-typescript.md

```diff
@@ -21,6 +21,9 @@ telemetryOptions: {
     metrics: {
       prometheus: { bindAddress: '0.0.0.0:9464' },
     },
-    logging: { forward: { level: 'DEBUG' } },
+    logging: {
+      forward: {},
```

Comment:
what does the empty object do?

Reply:
The empty object enables the option. That's unfortunate, but was already established previously, and we ended up deciding not to change this at this time.

Reply:
```suggestion
      forward: {}, // this enables log forwarding
```